python/private/pypi/pip_compile.bzl

```diff
@@ -63,7 +69,7 @@ def pip_compile(
         generate_hashes: whether to put hashes in the requirements_txt file.
         py_binary: the py_binary rule to be used.
         py_test: the py_test rule to be used.
-        requirements_in: file expressing desired dependencies. Deprecated, use src instead.
+        requirements_in: file expressing desired dependencies. Deprecated, use src or srcs instead.
```

Comment:
Should we deprecate `src` in favor of `srcs` (like how `requirements_in` was deprecated in favor of `src`)?

I figure this could be done in a follow-up PR, but it seems worth considering here.

Reply:
Yeah, I think deprecating `src` sounds like a good idea, but it can be done later.